### PR TITLE
style: count badge animation

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Add actionable count count initial animation.

# Changes

- apply `transition:scale` to the badge
- display after on mount to trigger animation after navigation.

# Tests

- manually.

# Todos

- [ ] Add entry to changelog (if necessary).

### CODE REVIEW

File: frontend/src/lib/components/proposals/ActionableProposalCountBadge.svelte [hunk 1]
```diff
@@ -1,10 +1,25 @@
...
+
+  // Always rerender to trigger animation start
+  let mounted = false;
+  onMount(() => (mounted = true));
 </script>
 
-<span data-tid="actionable-proposal-count-badge-component" class="tag"
-  >{count}</span
->
+{#if mounted}
```
Comment (dskloetd) [inline frontend/src/lib/components/proposals/ActionableProposalCountBadge.svelte:13]: Why is this line needed?
Without this, does the animation only trigger sometimes or never?

Comment (mstrasinskis) [reply frontend/src/lib/components/proposals/ActionableProposalCountBadge.svelte:13]: W/o `mounted` the animation isn't triggered after switching between Neurons and Proposals. The same Navigation is rendered on both pages but w/o the CountBadge on Neurons page.

**Review by dskloetd — APPROVED** [review]
